docker/init.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash -e 
+
+curl "https://awscli.amazonaws.com/awscli-exe-linux-x86_64.zip" -o "awscliv2.zip"
+unzip awscliv2.zip
```

Comment:
Shouldn't we do some checksum verification to ensure we are not pulling in malicious stuff?